src/main/java/com/amazon/opendistroforelasticsearch/security/dlic/rest/validation/AccountValidator.java

```diff
@@ -0,0 +1,13 @@
+package com.amazon.opendistroforelasticsearch.security.dlic.rest.validation;
```

Comment:
add copy right information and appropriate comments explaining what class or method is doing.